### PR TITLE
Vaishnavi_Assignment_20_60Min_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image
image
image
image

### CODE REVIEW

File: src/vaishnaviVJun20_arrayExamples/Armstrong.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+public class Armstrong {
+	
+	void findIfNumberIsArmstrong(int input) {
+		int Originalnumber = input;
+		int sum =0;
+		while(input>0) {
+			int num = input%10;
+			input = input/10;
+			int result = num*num*num;
```
Comment (AditiGJUN20) [inline src/vaishnaviVJun20_arrayExamples/Armstrong.java]: **Suggestion:**
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication of same number multiple times (to find square, cube and so on).
i.e. Math.pow((number % 10), 3))

File: src/vaishnaviVJun20_arrayExamples/Armstrong.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+			sum += result;
```
Comment (AditiGJUN20) [inline src/vaishnaviVJun20_arrayExamples/Armstrong.java]: On this line you can directly formulate the value of variable sum.
No need to declare extra variables and store values separately.(remove line 11 and 13)

File: src/vaishnaviVJun20_arrayExamples/ReverseInteger.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+//WAP to revere number (Consider Input as Integer not as String)
+
+public class ReverseInteger {
+	
+	void reverseIntegerNumber(int num) {
+		System.out.print("Reverse number is:");
+		while(num>0) {
+			int reverse = num%10;
+			num = num/10;
+			System.out.print(reverse);
```
Comment (AditiGJUN20) [inline src/vaishnaviVJun20_arrayExamples/ReverseInteger.java]: Here you are directly printing the reverse order of the given number, 
Implement integer reverse logic - which is addition of one more statement

**Review by AditiGJUN20 — CHANGES_REQUESTED** [review]

File: src/vaishnaviVJun20_arrayExamples/Armstrong.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20_arrayExamples/Armstrong.java]: --Implemented this logic using this function
--Thanks for the suggestion

File: src/vaishnaviVJun20_arrayExamples/Armstrong.java (hunk 2 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20_arrayExamples/Armstrong.java]: --Done

File: src/vaishnaviVJun20_arrayExamples/ReverseInteger.java (hunk 3 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20_arrayExamples/ReverseInteger.java]: --suggested changes done

**Review by AditiGJUN20 — APPROVED** [review]
Good job @VaishnaviVJun20 
Changes are made as per comments.

File: src/vaishnaviVJun20_arrayExamples/Armstrong.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package vaishnaviVJun20_arrayExamples;
+
+//WAP to check if a number is Armstrong or not.
+
+public class Armstrong {
+	
+	void findIfNumberIsArmstrong(int input) {
+		int Originalnumber = input;
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20_arrayExamples/Armstrong.java:8]: Variable name started with capital letter !!!

File: src/vaishnaviVJun20_arrayExamples/DuplicateNumbers.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package vaishnaviVJun20_arrayExamples;
+
+//Find duplicate number on Integer array in Java?
+
+public class DuplicateNumbers {
+	
+	void findDuplicateNumber(int[] array) {
+		for(int index=0;index<array.length;index++) {
+			for(int i = index+1;i<array.length;i++) {
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20_arrayExamples/DuplicateNumbers.java:9]: Instead of i use innerIndex.

**Pull request merged**